Problem: 服务器:{HOSTNAME1}发生: {TRIGGER.NAME}故障!
告警主机: {HOSTNAME1}
告警时间: {EVENT.DATE} {EVENT.TIME}
告警等级: {TRIGGER.SEVERITY}
告警信息: {TRIGGER.NAME}
告警项目: {TRIGGER.KEY1}
问题详情: {ITEM.NAME}: {ITEM.VALUE}
当前状态: {TRIGGER.STATUS}: {ITEM.VALUE1}
事件ID: {EVENT.ID}

OK: 服务器:{HOSTNAME1}: {TRIGGER.NAME}已恢复!
告警主机: {HOSTNAME1}
告警时间: {EVENT.DATE} {EVENT.TIME}
恢复时间: {EVENT.RECOVERY.DATE} {EVENT.RECOVERY.TIME}
持续时间: {EVENT.AGE}
告警等级: {TRIGGER.SEVERITY}
告警信息: {TRIGGER.NAME}
告警项目: {TRIGGER.KEY1}
问题详情: {ITEM.NAME}: {ITEM.VALUE}
当前状态: {TRIGGER.STATUS}: {ITEM.VALUE1} : {ITEM.VALUE1}
事件ID: {EVENT.ID}


服务器: {HOST.NAME}: 报警确认
确认人: {USER.FULLNAME} 
时间: {ACK.DATE} {ACK.TIME} 
确认信息如下:
"{ACK.MESSAGE}"
问题服务器IP: {HOSTNAME1}
问题ID: {EVENT.ID}
